app/js/third_party/jsspeechrecogniser.js

```diff
@@ -0,0 +1,717 @@
+/**
```

Comment:
The way we're handling external dependencies is usually the following:
* Install via npm (In this case, it's not published on npm so the syntax to use in `package.json` is something like `"JsSpeechRecognizer": "github:dreamdom/JsSpeechRecognizer#master"` (not sure if the master is required)
* Copy the files you need from `node_modules` to `app/js/components` using the `copy-app-common` gulp task (see  `copy-app-dev` and `copy-app-production` tasks if you need to copy different files on dev and prod environments)
* Ideally load the file using es2015 modules, or via a `<script>` tag if not compatible

This avoid checking in external files into the tree.
Sorry for not having documenting this part.

Does it make sense to you?